### DISCUSSION TITLE
en/about/

### DISCUSSION THREAD

**Discussion by giscus** [nhtithy/nhtithy.github.io#5] [Ideas]
# en/about/

Naznin Hasan Tithy Student, Photogrpaher, Desinger.
nazninhasan.tithy@gmail.com | My website Currently Studying Honours
Specialized in Photogrpahy, Cooking.
Education 2023-Now Chuadanga Govt. college, Chuadanga.
BA 2020 - 2022 Chuadanga Govt. college, Chuadanga.
HSC 2016 - 2020 Jhinuk high school, Chuadanga
The very Begining Language Language Proficiency 1) English Fluent 2) HIndi Fluent 3) Bengali Native 

https://nhtithy.github.io/en/about/

**Comment by mrsnailo** [comment]
Hi
